metadata-ws/src/main/resources/db/changelog/db.changelog-1.0.0.xml

```diff
@@ -0,0 +1,339 @@
+	</changeSet>
+	<changeSet author="metauser (generated)" id="1569329013185-46">
+		<addForeignKeyConstraint baseColumnNames="samples_id" baseTableName="analysis_samples" constraintName="fkiuqiceb6v0uhh3qhpmfv4ma2b" deferrable="false" initiallyDeferred="false" onDelete="NO ACTION" onUpdate="NO ACTION" referencedColumnNames="id" referencedTableName="sample" validate="true"/>
+	</changeSet>
+	<changeSet author="metauser (generated)" id="1569329013185-47">
+		<addForeignKeyConstraint baseColumnNames="sample_id" baseTableName="sample_taxonomies" constraintName="fkm2t1l9yk59ml3yt4qmoviqqda" deferrable="false" initiallyDeferred="false" onDelete="NO ACTION" onUpdate="NO ACTION" referencedColumnNames="id" referencedTableName="sample" validate="true"/>
+	</changeSet>
+	<changeSet author="metauser (generated)" id="1569329013185-48">
+		<addForeignKeyConstraint baseColumnNames="order_id" baseTableName="taxonomy" constraintName="fkmd0urpfct62pdmev27e190g7v" deferrable="false" initiallyDeferred="false" onDelete="NO ACTION" onUpdate="NO ACTION" referencedColumnNames="taxonomy_id" referencedTableName="taxonomy" validate="true"/>
+	</changeSet>
+	<changeSet author="metauser (generated)" id="1569329013185-49">
+		<addForeignKeyConstraint baseColumnNames="study_id" baseTableName="analysis" constraintName="fkonf7hcukl3xwb7ys49peax08s" deferrable="false" initiallyDeferred="false" onDelete="NO ACTION" onUpdate="NO ACTION" referencedColumnNames="id" referencedTableName="study" validate="true"/>
+	</changeSet>
+	<changeSet author="metauser (generated)" id="1569329013185-50">
+		<addForeignKeyConstraint baseColumnNames="taxonomies_id" baseTableName="sample_taxonomies" constraintName="fkpsni44vdpwk4xxppgppq7c4m3" deferrable="false" initiallyDeferred="false" onDelete="NO ACTION" onUpdate="NO ACTION" referencedColumnNames="id" referencedTableName="taxonomy" validate="true"/>
+	</changeSet>
+	<changeSet author="metauser (generated)" id="1569329013185-51">
+		<addForeignKeyConstraint baseColumnNames="analysis_id" baseTableName="analysis_files" constraintName="fkq67f06oxvmdj36hdu363cpg1e" deferrable="false" initiallyDeferred="false" onDelete="NO ACTION" onUpdate="NO ACTION" referencedColumnNames="id" referencedTableName="analysis" validate="true"/>
+	</changeSet>
+	<changeSet author="metauser (generated)" id="1569329013185-52">
+		<addForeignKeyConstraint baseColumnNames="study_id" baseTableName="study_child_studies" constraintName="fktfwfp9t1op8ku2pcjrw8c80nw" deferrable="false" initiallyDeferred="false" onDelete="NO ACTION" onUpdate="NO ACTION" referencedColumnNames="id" referencedTableName="study" validate="true"/>
+	</changeSet>
+
+	<changeSet id="tag-1.0" author="metauser">
+		<tagDatabase tag="1.0" />
+	</changeSet>
+</databaseChangeLog>
```

Comment:
Please add a file to the `docs/` folder explaining:
- How Liquibase works for our project (briefly)
- How to generate the new changelog for newly released version
- How to update the main `liquibase-changelog.xml`

Reply:
I feel confluence page may be a good choice here.